assets/js/modules/analytics/util/index.js

```diff
@@ -620,7 +620,7 @@ export const getTopPagesReportDataDefaults = () => {
  */
 export const parseTotalUsersData = ( data ) => {
 	return {
-		totalUsers: data?.[0]?.data?.totals?.[0]?.values?.[0],
-		previousTotalUsers: data?.[0]?.data?.totals?.[1]?.values?.[0],
+		totalUsers: data?.[ 0 ]?.data?.totals?.[ 0 ]?.values?.[ 0 ],
+		previousTotalUsers: data?.[ 0 ]?.data?.totals?.[ 1 ]?.values?.[ 0 ],
```

Comment:
Did you see an error for this? I noticed that this code block in particular was failing to be caught by ESLint with the current packages we have now.

Reply:
@aaemnnosttv Yes, eslint was complaining about it :(

Reply:
That's good! 😄